.gitignore

```diff
@@ -18,3 +18,4 @@ app/lib/core/Parsers/htmlpurifier/standalone/HTMLPurifier/DefinitionCache/Serial
 app/lib/core/Parsers/dompdf/lib/fonts/*fm.php
 app/log/log*.txt
 themes/default/views/bundles/inspector_info.php
+/install/profiles/sql
```

Comment:
shoudn't it be `install/profiles/sql`? to be consistent with the rest of the file?